src/utils/promise.js

```diff
@@ -1,4 +1,6 @@
 // This is CommonJS because lie is an external dependency, so Rollup
 // can just ignore it.
-require('lie/polyfill');
+if (typeof Promise === 'undefined' && typeof require !== 'undefined') {
```

Comment:
the reason this won't fix the underlying issue is that it will still fail on non-Promise browsers, right?